nativelib/src/main/scala/scala/scalanative/runtime/ieee754tostring/ryu/RyuRoundingMode.scala

```diff
@@ -35,34 +35,19 @@ package scala.scalanative
 package runtime
 package ieee754tostring.ryu
 
-object RyuRoundingMode extends Enumeration {
```

Comment:
Friends don't let friends use `scala.Enumeration`. This used to add a few hundred KB in transitive dependencies.